<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightControlWindowClass</class>
 <widget class="QMainWindow" name="LightControlWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>256</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Light Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="LightControl.qrc">
    <normaloff>:/LightControlWindow/icon.ico</normaloff>:/LightControlWindow/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>COM</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Select COM port</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Relay</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="led1Label">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: gray;
border-radius: 5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="relay1PushButton">
           <property name="text">
            <string>Relay 1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="led2Label">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: gray;
border-radius: 5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="relay2PushButton">
           <property name="text">
            <string>Relay 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="led3Label">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: gray;
border-radius: 5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="relay3PushButton">
           <property name="text">
            <string>Relay 3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="led4Label">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: gray;
border-radius: 5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="relay4PushButton">
           <property name="text">
            <string>Relay 4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Module id:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="idLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Firmware v:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="versionLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LightControl.qrc"/>
 </resources>
 <connections/>
</ui>
